src/Console/Locale/EnableLocaleCommand.php

```diff
@@ -219,6 +219,14 @@ private function askToInstall()
             return false;
         }
 
+        $this->formatter->writeln(
+            '<info>Before you can enable a new locale you need to authenticate to be able to create the pages</info>'
+        );
+
+        while (!Authentication::loginUser($this->formatter->ask('Login'), $this->formatter->askHidden('Password'))) {
```

Comment:
@bjorvack in english please, this is a global effort ;)

As for your remark: You can only access this command if you have access to the source or the server so I think it's not necessary to prevent bruteforcing